Dockerfile

```diff
@@ -9,6 +9,6 @@ COPY . /usr/src/app
 # Install app dependencies
 RUN npm install
 
-EXPOSE 5000
+EXPOSE 3000
```

Comment:
We should expose 5000 for production. See https://github.com/asyncapi/editor/blob/master/config/common.yml#L9

My suggestion is that you leave this one as 5000 and change the config file default value here: https://github.com/asyncapi/editor/blob/master/config/common.yml#L3.

And also change the README you added to point to 5000.

Reply:
done.  updated to use port 5000.  also sent NODE_ENV default in dockerfile to indicate which environment settings it will use.